Formula/jsign.rb

```diff
@@ -0,0 +1,50 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign/archive/refs/tags/5.0.tar.gz"
+  sha256 "7b77a12aaea4f404e7b243bd58cfde485eb03b44219e128338c9fe6617ad1fa1"
+  license "Apache-2.0"
+  head "https://github.com/ebourg/jsign.git", branch: "master"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17" # The build fails with more recent JDKs
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@17"].opt_prefix
+    system("mvn", "--batch-mode", "package", "--projects", "jsign-core,jsign-cli,jsign-ant,jsign", "-DskipTests",
+           "-Djdeb.skip", "-Dmaven.javadoc.skip")
+
+    # Fetch the version from the pom (required to build from HEAD)
+    require "rexml/document"
+    pom = REXML::Document.new(File.new("pom.xml"))
+    version = REXML::XPath.first(pom, "string(/pom:project/pom:version)", "pom" => "http://maven.apache.org/POM/4.0.0")
+
+    libexec.install Dir["jsign/target/jsign-#{version}.jar"]
```

Comment:
Couldn't we just do this?
```suggestion
    jsign_jar = testpath.glob("jsign/target/jsign-#{version}.jar").first
    libexec.install jsign_jar
```

Reply:
No that doesn't work when building from HEAD, the version has to be extracted from the `pom.xml` file.

Reply:
Sorry, there was a typo in my suggestion. Please check again -- it seems like it should work for HEAD.

Reply:
There are several files matching the `jsign-*.jar` pattern in this directory, it's not guaranteed to catch the right one.

Reply:
Ok, that's fine, but we should avoid using `Dir` since that will silently do nothing if the file you're looking for is not there:
```suggestion
    # Fetch the version from the pom (required to build from HEAD)
    require "rexml/document"
    pom = REXML::Document.new(File.new("pom.xml"))
    jar_version = REXML::XPath.first(pom, "string(/pom:project/pom:version)", "pom" => "http://maven.apache.org/POM/4.0.0")

    libexec.install "jsign/target/jsign-#{jar_version}.jar"
```

Reply:
`Dir` has been removed